### PR TITLE
New GMDN import scripts

### PR DESCRIPTION
**Pull request by alaric-rd**

### CODE REVIEW

File: gmdn/parse.py [hunk 1]
```diff
@@ -14,15 +16,67 @@ def startElement(self, name, attrs):
...
-            print(json.dumps(self._record))
+            fp.write(json.dumps(self._record))
+            fp.write("\n")
+        else:
+            self._record[name] = ''.join(self._buffer)
+
+    def characters(self, data):
+        self._buffer.append(data)
+
+xml.sax.parse("newData/gmdnData24_6.xml", DataHandler())
```
Comment (rossjones) [inline gmdn/parse.py]: Should we document that you need to put the data into a folder called newData?

**Review by rossjones — APPROVED** [review]
LGTM, minor issue in that it didn't work for me because I didn't have the XML in a newData/ directory so it might be worth adding a README?

File: gmdn/parse.py (hunk 1 above)
Comment (alaric-rd) [reply gmdn/parse.py]: I moved it, seemed a cleaner fix!

**Review by rossjones — APPROVED** [review]
👍🏻

**Pull request merged**